operations-stdlib/src/commonTest/kotlin/string/TrimTest.kt

```diff
@@ -0,0 +1,133 @@
+import JsonLogicResult.Failure
+import JsonLogicResult.Success
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.datatest.withData
+import io.kotest.matchers.shouldBe
+import string.Trim
+
+class TrimTest : FunSpec({
```

Comment:
adding some more test cases where you use **data** and/or complex expressions with some other operations(var for example) would be great

Reply:
I think its fine now, tests for operators are almost infinite. The more the better :D